Makefile

```diff
@@ -74,8 +74,7 @@ $(error Unsupported platform to compile for)
 endif
 
 GO_BUILD       = env GOOS=$(PLATFORM) GOARCH=$(ARCH) go build -i $(GOFLAGS) \
-                   -ldflags "-X $(SC_PKG)/pkg.VERSION=$(VERSION)" \
-                   -ldflags "$(BUILD_LDFLAGS)"
+                   -ldflags "-X $(SC_PKG)/pkg.VERSION=$(VERSION) $(BUILD_LDFLAGS)"
```

Comment:
you can still split flags on separate lines with `\` (with extra tab for readability) to make it easier to read